composer.json

```diff
@@ -29,6 +29,7 @@
     "source": "https://github.com/owncloud/gallery"
   },
   "require": {
+    "symfony/yaml": "~2.6"
```

Comment:
> Are there other `symfony` components that are needed at run-time?

`symfony/yaml` should be enough - that is what is needed for the cnf file parsing. It will pull in whatever it then needs.